dpsim/src/Simulation.cpp

```diff
@@ -379,7 +379,7 @@ void Simulation::stop() {
 }
 
 Real Simulation::next() {
-	if (mTime < (**mFinalTime + **mTimeStep))
+	if (mTime < **mFinalTime + 1e-12)
```

Comment:
This number should specified via a #define directive for example in `dpsim/include/dpsim/Definitions.h`